modules/viewing-the-service-logs-cli.adoc

```diff
@@ -0,0 +1,143 @@
+// Module included in the following assemblies:
+//
+// * osd_cluster_admin/osd_logging/osd-accessing-the-service-logs.adoc
+// * rosa_cluster_admin/rosa_logging/rosa-accessing-the-service-logs.adoc
+
+:_content-type: PROCEDURE
+[id="viewing-the-service-logs-cli_{context}"]
+= Viewing the service logs by using the CLI
+
+You can view the service logs for 
+ifdef::openshift-dedicated[]
+{product-title} 
+endif::openshift-dedicated[]
+ifdef::openshift-rosa[]
+{product-title} (ROSA) 
+endif::openshift-rosa[]
+clusters by using the {cluster-manager} CLI (`ocm`).
```

Comment:
ocmcli is not an official tool, should it be listed here?

Reply:
@xueli181114 I have removed the content related to the OpenShift Cluster Manager CLI from this PR. Please let me know if the PR now looks ok. Thanks!

Reply:
I have also removed any other OCM CLI references from the OSD and ROSA libraries through https://github.com/openshift/openshift-docs/pull/47385.

Reply:
Thanks. I have checked the preview. No other problems from me.

Reply:
Thank you!